server/game/cards/08.5-TFM/TheSongOfTheSeven.js

```diff
@@ -0,0 +1,38 @@
+const DrawCard = require('../../drawcard.js');
+
+class TheSongOfTheSeven extends DrawCard {
+    setupCardAbilities(ability) {
+        this.reaction({
+            max: ability.limit.perRound(1),
+            when: {
+                onDominanceDetermined: event =>
+                    this.controller !== event.winner &&
+                    event.winner.faction.power >= 2 &&
```

Comment:
If event.winner is undefined (in case of a dominance tie), this will likely crash, so you'll need to check for that first. Also, this card should still trigger if the opponent has a single power on their house card.